apps/web/src/pages/tenants/Tenants.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PageContainer from '../../components/layout/components/PageContainer';
+import PageHeader from '../../components/layout/components/PageHeader';
+import { TenantsList } from './components/list/TenantsList';
+
+export function Tenants() {
```

Comment:
Not mandatory but I created this one because i thought it will be a good starting point for the `tenant page`(discovery in the code), additionally in order to encapsulate the page itself, for example as we had in the integration page initially it was a page of cards but now we refactored it to a page with a list. in the next iteration, I want to create additional middleware between Tenant and TenantsList components with the feature flag.

Reply:
sounds good. Lets go for it

Reply:
ok so i will merge it and if it will be too much i will revert it, should not be too much work.